core/storage/suggestion/gae_models_test.py

```diff
@@ -43,8 +43,10 @@ class SuggestionModelUnitTests(test_utils.GenericTestBase):
     """Tests for the suggestionModel class."""
 
     score_category: str = (
-        suggestion_models.SCORE_TYPE_TRANSLATION +
-        suggestion_models.SCORE_CATEGORY_DELIMITER + 'English')
+        '%s%s%s' % (
+        suggestion_models.SCORE_TYPE_TRANSLATION,
```

Comment:
Indent this line and the next by 4 from the previous line, since the ( introduces a new indentation scope.